qml/StatisticsChartView.qml

```diff
@@ -38,6 +38,12 @@ ChartView {
     property var tooltipItem: tooltip
     property bool manuallySetAxes: false
 
+    // intValdiator: IntValidator {}
+    property real y_MAX_DEFAULT: dynamicData.getMinReal()
```

Comment:
NIT: Is this really needed? Have you checked if min and max values could be used from javascript?